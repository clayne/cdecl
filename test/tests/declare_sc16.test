cdecl @ /dev/null @ @ declare x as signed char16_t @ 65
