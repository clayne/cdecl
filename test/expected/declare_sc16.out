declare x as signed char16_t
        ^
9: error: "signed char16_t" is illegal for built-in type
